javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,10 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  var sufArray = [];
+  var newWord = words[0] + suffix;
```

Comment:
I'm not sure why you would do this before the loop begins, this can be simplified to just do the same work on each iteration of the loop.

Reply:
and again..